plugins/data-management/src/ucsc-trackhub/model.js

```diff
@@ -22,6 +23,20 @@ export default function (pluginManager) {
         configuration: ConfigurationReference(configSchema),
         type: types.literal('UCSCTrackHubConnection'),
       })
+      .views(self => ({
+        get canConfigure() {
```

Comment:
where is this canConfigure used? It might be good to avoid knowing about adminMode stuff in the plugins if possible

Reply:
I just added it cause I saw with session tracks they had a canConfigure so I thought it might be needed for sessionConnections. If not necessary to work I can remove it

Reply:
Ya, I think it probably can be removed. I think it'd otherwise have to be duplicated across all types of connections if this was more general.